plugins/filter/json_diff.yml

```
@@ -0,0 +1,51 @@
+---
+# Copyright (c) Stanislav Meduna (@numo68)
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+DOCUMENTATION:
+  name: json_diff
+  short_description: Create a JSON patch by comparing two JSON files
+  description:
+    - This filter compares the input with the argument and computes a list of operations
+      that can be consumed by the P(community.general.json_patch_recipe) to change the input
+      to the argument.
+  requirements:
+    - jsonpatch (on the controller)
+  version_added: 10.3.0
+  author:
+    - Stanislav Meduna (@numo68)
```

```suggestion
    - Stanislav Meduna (@numo68)
  positional: ...
```
You need to list the positional arguments in order here. Otherwise the users have no clue in which order the options must be supplied in case they're not keyword options.